documentation/sphinx/source/developer-guide.rst

```diff
@@ -756,6 +756,118 @@ If you only need to detect the *fact* of a change, and your response doesn't dep
 
 .. _developer-guide-peformance-considerations:
 
+
+Special keys
+============
+
+Keys starting with the bytes ``\xff\xff`` are called "special" keys, and they are materialized when read. :doc:`\\xff\\xff/status/json <mr-status>` is an example of a special key.
+As of api version 630, additional features have been exposed as special keys and are available to read as ranges instead of just individual keys. Additionally, the special keys are now organized into "modules".
+
+Modules
+-------
+
+A module is loosely defined as a key range where a user can expect similar behavior from reading any key in that range.
+By default, users will see a ``special_keys_no_module_found`` error if they read from a range not contained in a module.
+This is likely to be an error since the read would always return an empty set of keys and can't be interesting (maybe it was a typo?).
```

Comment:
I didn't get the meaning or purpose of this sentence, although I understand every word of it.

Reply:
I guess you want to explain the reason of the behavior. Since the audience is the user of FDB, we probably don't need to explain that.

Reply:
How about change Line 771 to `The error indicates the read would always return an empty set of keys if it proceeded. This could be caused by typo in the keys to read.`

Reply:
IIUC, this is trying to explain the reason to introduce this error. IMHO, using a statement feels better than a question.